drivers/bluetooth/hci/Kconfig

```diff
@@ -102,23 +112,6 @@ config BT_STM32_IPM_RX_STACK_SIZE
 	depends on BT_STM32_IPM
 	default 512
 
-config BT_RPMSG_NRF53
-	bool "nRF53 configuration of RPMsg"
-	default y if SOC_NRF5340_CPUAPP
-	depends on BT_RPMSG
-	select RPMSG_SERVICE
-	help
-	  Enable RPMsg configuration for nRF53. Two channels of the IPM driver
-	  are used in the HCI driver: channel 0 for TX and channel 1 for RX.
-
-if BT_RPMSG_NRF53
-
-choice RPMSG_SERVICE_MODE
-	default RPMSG_SERVICE_MODE_MASTER
-endchoice
-
-endif # BT_RPMSG_NRF53
-
```

Comment:
Does this change mean that RPMSG is not automatically enabled for the nRF5340 CPUAPP?

Reply:
No, it does not. I migrated the HCI driver to a transport module (IPC service) with a better abstraction. It is no longer necessary to have SoC specific code.